apps/frontend/client/vite.config.ts

```
@@ -45,6 +48,71 @@ function isIgnoredWarning(msg: string): boolean {
+//   unset                            → exclude iff the VITE BUILD MODE is
+//                                       'production' (set by vite.config.ts)
+//
+// NODE_ENV is deliberately NOT consulted: moon sets NODE_ENV=production for
+// every build task regardless of target mode, so using it here would strip
+// the (dev) sandbox routes from test/QA builds (M4). The vite mode is the
+// single source of truth.
+// ---------------------------------------------------------------------------
+const buildMode = process.env.AIKAMI_BUILD_MODE;
+const devGateOverride = process.env.AIKAMI_INCLUDE_DEV_ROUTES;
+const isProductionBuild = buildMode === 'production';
+let includeDevRoutes;
```

Biome (2.5.7)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)